source/changelog.txt

```diff
@@ -12,6 +12,24 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v1.10.1
+-------
+
+*Released June 21, 2023*
+
+- Upgrades to Node.js version 16.20.1. Node 16.20.1 addresses security
+  issues with OpenSSL. For more information, see
```

Comment:
```suggestion
  issues. For more information, see
```

Not just OpenSSL, it also includes security fixes for other dependencies as well as Node.js itself.